tests/lib/network_helper.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Jan 23 10:45:21 2019
+
+@author: gerben
+"""
+
+from netaddr import *
+import socket
+import struct
+import re
+class mac_custom(mac_unix):
+    pass
+
+def mac_format(mac_addr):
+    mac_tmp = re.sub("[\s+\.\!\-\:\/_,$%^*(+\"\')]+|[+()?【】“”！，。？、~@#￥%……&*（）]+',","",mac_addr)
+    mac_custom.word_fmt = '%.2X'
+    mac = EUI(mac_tmp,dialect=mac_custom)
+    return mac
+
+'''IP Decimal Type 2, 10, 16
+e.g.
+ip2addr = '10101100000100000000000100000001'
+ip10addr = '2886729985'
+ip16addr = 'AC100101'
+ip = ipv4_format(ip$addr,$decimal)
+also turn to 172.16.1.1
+'''
+def ipv4_format(ipv4_str,ip_decimal):
```

Comment:
I can't quite tell the purpose of this function can you please say what you are converting it too?

ip_to_dotted_decimal()

Secondly, why not use the IP address library the rest of the boardfarm is trying to standardize on?

https://docs.python.org/3/howto/ipaddress.html

Reply:
Lib only to check, I also need to deal with ip-address again. likes:
ipaddress.ip_network(3221225984)
get 192.0.2.0/32
I need to delete '/32'.